open-metadata-implementation/access-services/docs/design/README.md

```diff
@@ -204,18 +204,26 @@ implementing the OMAS one scenario at a time.
   
   Link to the the top-level README.md for the OMAS's server module (***omas-name*/*omas-name*-server/README.md**).
   
-  Implement the ***omas-name*/*omas-name*-server-spring module for the OMAS.  This contains the sever-side
+  Implement the ***omas-name*/*omas-name*-server-spring** module for the OMAS.  This contains the sever-side
   resource classes that support the REST API.
   
-  Add the ***omas-name*/*omas-name*-server-spring module to the `pom.xml` file for the 
+  Add the ***omas-name*/*omas-name*-server-spring** module to the `pom.xml` file for the
   [OMAG Server Chassis](../../../server-chassis/server-chassis-spring).
 
 * Design and implement the event handling.
 
-* Implement the FVT and run it.
-
+  Log each event that is picked up (inbound) to be processed by the OMAS into its audit log using the `EVENT` severity.
+  This ensures a record is kept of the asynchronous processing done by the OMAS.
```

Comment:
I wasn't sure what we want to recommend for outbound -- eg. if an OMAS produces an outbound event that nothing else processes then is it important to know that?  (If another services makes use of that event, then per this guidance that other services' inbound processing should log that it is using the event...)

On the other hand, from an auditing perspective, it would likely be good to be able to answer the question "where did the inbound event that I'm processing come from" in which case having the OMAS log its outbound-generated events also seems like a good idea.

Suggestion / preference?

Reply:
I think we shuld recommend logging both in and out.  One of the hardest things to diagnose is when something does not happen that should have :)

Reply:
Makes sense -- latest commit adds a line immediately after the inbound statement as follows: "Also log each event that is produced (outbound) by the OMAS into its audit log using the `EVENT` severity."